server/src/serializers/ActivitySerializer.js

```diff
@@ -0,0 +1,27 @@
+class ActivitySerializer {
+    static getSummaryOfOne(activity){
+        const acceptedAttributes = ["id", "name", "description"]
+        const serializedActivity = {}
+
+        for(const attribute of acceptedAttributes){
+            serializedActivity[attribute] = activity[attribute]
+        }
+        return serializedActivity
+    }
```

Comment:
this `getSummaryOfOne` is not being used, we should either remove it, or make use of it to help serialize individual Activities in the method below (since they do the same logic)